rand_distr/benches/distributions.rs

```diff
@@ -6,18 +6,17 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+#![feature(custom_inner_attributes)]
```

Comment:
Why is this feature added?

Reply:
Using `rustc 1.46.0-nightly (8ac1525e0 2020-07-07)` if I remove this I get:
```
$ cargo +nightly bench --features=nightly
   Compiling rand v0.7.3 (/home/jon/github/jongiddy/rand)
error[E0658]: non-builtin inner attributes are unstable
  --> benches/distributions.rs:12:1
   |
12 | #![rustfmt::skip]
   | ^^^^^^^^^^^^^^^^^
   |
   = note: see issue #54726 <https://github.com/rust-lang/rust/issues/54726> for more information
   = help: add `#![feature(custom_inner_attributes)]` to the crate attributes to enable
```